tests/testTriggers.go

```diff
@@ -1240,6 +1259,22 @@ func TriggerVolumeClone(contexts *[]*scheduler.Context, recordChan *chan *EventR
 			})
 			for _, vol := range appVolumes {
 				var clonedVolID string
+
+				// Skip clone trigger for Pure FB volumes
+				isPureFileVol, err := Inst().V.IsPureFileVolume(vol)
+				if err != nil {
+					logrus.Errorf("Checking pure file volume is failing with error: %v", err)
```

Comment:
It will be helpful to print the volume name/ID along with the error here.

Reply:
Incorporated !!